core/src/main/scala/org/apache/spark/status/AppStatusListener.scala

```diff
@@ -107,6 +111,8 @@ private[spark] class AppStatusListener(
 
   override def onOtherEvent(event: SparkListenerEvent): Unit = event match {
     case SparkListenerLogStart(version) => sparkVersion = version
+    case MiscellaneousProcessInfoEvent(time, cores, memory, hostName, urlInfo) =>
+      updateAMInfoInLiveProcess(time, cores, memory, hostName, urlInfo.toMap)
```

Comment:
rename to be something like updateMiscProcessInfo.

Also any Reason you didn't create an onMiscellaneousProcessAdded type function?

Reply:
Added the code change for making use on onMiscellaneousProcessAdded type function having MiscellaneousProcessInfoEvent in the same manner as its done in  SparkListenerSQLExecutionStart , SparkListenerSQLAdaptiveExecutionUpdate  etc